utils/rpms/daos.spec

```diff
@@ -65,6 +65,7 @@ BuildRequires: protobuf-c-devel
 BuildRequires: lz4-devel
 BuildRequires: capstone-devel
 %endif
+BuildRequires: libaio-devel
```

Comment:
This change requires bumping the release and a changelog entry.

Reply:
Thank you! Yes. I will add it later once current CI tests pass and get more feedback

Reply:
Release number was bumped up.